src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -75,9 +75,9 @@ export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
         <div className={styles.tagAndLabelContainer}>
           {category === 'InternalNews' ? (
             <Category category={CONTENT_CATEGORIES.NEWS} />
-          ) : (
+          ) : category !== 'LandingPage' ? (
             <Tag className={`${tagStyles.Category}`}>{category}</Tag>
-          )}
+          ) : null}
```

Comment:
When looking at an article nested under a landing page, the `LandingPage` category was being shown. I thought that might be confusing for the user so I added this to hide it.